addons/web/static/src/views/fields/properties/kanban_properties_field.scss

```diff
@@ -1,3 +1,11 @@
 .o_kanban_property_field {
     max-width: 300px;
 }
+
+.form-check{
```

Comment:
Doesn't that apply to every single form-check class in the entire world? Would need to be included inside a more specific rule ideally.